### PR TITLE
Modal toast grid

### PR DESCRIPTION
**Pull request by yashodaAgarwal**
Please review my components

### CODE REVIEW

**Comment by PuneetP16** [comment]
image
Contrast ratio could be improved to increase readability.

File: style/modal.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    pointer-events: auto;
+    opacity: 1;
+    z-index: 5;
+}
+.modal{
+    background-color: var(--white-color);
+    border-radius: 5px;
+    box-shadow: 0 2px 4px var(--shadow-color);
+    padding: 30px 3.2rem;
+    width: 600px;
```
Comment (PuneetP16) [inline style/modal.css]: Could use rems to make it responsive with respect to root font-size.

**Review by PuneetP16 — APPROVED** [review]

**Comment by ArpitPatni** [comment]
Looks good to merge!

File: style/modal.css (hunk 1 above)
Comment (yashodaAgarwal) [reply style/modal.css]: done

**Pull request merged**